test/expected/privileged_role.out

```diff
@@ -176,11 +176,11 @@ alter role authenticator nologin;
 ERROR:  "authenticator" is a reserved role, only superusers can modify it
 \echo
 
--- privileged_role cannot manage [no]superuser attribute
+-- privileged_role cannot manage superuser attribute
 create role r superuser;
 ERROR:  permission denied to create role
 DETAIL:  Only roles with the SUPERUSER attribute may create roles with the SUPERUSER attribute.
-create role r;
+create role r nosuperuser;
```

Comment:
can we add a separate test case for this vs. editing existing ones? (just copy/paste the thing in between `\echo`s)

Reply:
I think the in-place update is more consistent with other tests for role attributes. For eg. see how we currently test bypassrls

https://github.com/supabase/supautils/pull/87/files#diff-48cbc0f6115c1290715458f4c0fa027021fdf1147b43eaf3f15f95f0ffd88e5eR37-R51

Reply:
sure, then let's do extra `create role`s instead of editing existing lines

Reply:
Feel free to add it if that's a blocker for you. Personally I don't think it's necessary because `create role tmp` already covers it https://github.com/supabase/supautils/blob/858d6c39a2afa0ce1b59e2429cf4a7d698bf7c4f/test/expected/privileged_role.out#L148-L151

Reply:
Updated the test